.github/unittest/action.yml

```diff
@@ -0,0 +1,26 @@
+name: Run .NET unit tests
+description: Runs .NET unit tests under a specified configuration
+
+inputs:
+  project:
+    default: ""
+    description: The test project
+    required: true
+  configuration:
+    default: Debug
+    description: The build configuration
+    required: false
+
+runs:
+  using: composite
+  steps:
+  - run: dotnet msbuild "${{ inputs.project }}"
+      -t:Rebuild
+      -p:WarningLevel=0
+      -p:Configuration=${{ inputs.configuration }}
+    shell: bash
+  - run: dotnet test "${{ inputs.project }}"
+      --no-build
+      --logger "GitHubActions;report-warnings=true"
+      --configuration "${{ inputs.configuration }}"
+    shell: bash
```

Comment:
and honesly not sure if we need a special action.yml for unit tests when it's just a matter of running `dotnet test`...

Reply:
Now that we create a report for unit tests as well and also run Analyzer tests, I think it's worth separating into its own action.